src/adafruit_blinka/microcontroller/mcp2221/mcp2221.py

```diff
@@ -64,6 +94,46 @@ def __init__(self):
         for pin in range(4):
             self.gp_set_mode(pin, self.GP_GPIO)  # set to GPIO mode
             self.gpio_set_direction(pin, 1)  # set to INPUT
+        return self
+
+    @staticmethod
+    def available_paths(*, require_mcps=False):
```

Comment:
For this method I considered a few different options:
* a read-only `@property`, but in order to work in `__new__` it must work without instance (and `@property` requires the instance);
* a stand-alone function, but it would need to be imported separately, and having it as a method is handy.

Regarding the name:
* I used `paths` because that's the name used by `hid.enumerate`.  Perhaps `addresses` or `devices` might be better.
* I used `required_mcps`, but `required_mcp2221s` might be better even if longer.  Other options (like `raise_if_no_mcp2221s`) feel too verbose.